src/Markdown.MAML/Renderer/Markdownv2Renderer.cs

```diff
@@ -283,17 +283,7 @@ private void AddParameter(MamlParameter parameter, MamlCommand command)
 
                 AppendYamlKeyValue(MarkdownStrings.Type, parameter.Type);
 
-                string parameterSetsString;
-                if (command.Syntax.Count == 1 || set.Item1.Count == command.Syntax.Count)
-                {
-                    // ignore, if there is just one parameter set
-                    // or this parameter belongs to All parameter sets, use (All)
-                    parameterSetsString = ModelTransformerVersion2.ALL_PARAM_SETS_MONIKER;
-                }
-                else
-                {
-                    parameterSetsString = JoinWithComma(set.Item1);
-                }
+                string parameterSetsString = JoinWithComma(set.Item1);
```

Comment:
We need to distinguish 2 cases, as we discussed in https://github.com/PowerShell/platyPS/issues/401#issuecomment-430472533

Reply:
Take a look at this example

```
function foo
{
    param(
        [Parameter(ParameterSetName="Default")]
        [Parameter(ParameterSetName="p1")]
        [string]$bar,
        [Parameter(ParameterSetName="p1", Mandatory=$true)]
        [string]$foo,
        [string]$all
    )
}
Get-Help -Full foo
```

```
...
    -all <string>
        
        Required?                    false
        Position?                    Named
        Accept pipeline input?       false
        Parameter set name           (All)
        Aliases                      None
        Dynamic?                     false
        
    -bar <string>
        
        Required?                    false
        Position?                    Named
        Accept pipeline input?       false
        Parameter set name           p1, Default
        Aliases                      None
        Dynamic?                     false
...
```

Note
a) For `-all ` the `Parameter set name           (All)` not `__AllParameterSets`
b) While the `-bar` has `p1, Default`, the `-all` has `(All)`. Both of them are applied to all parameter sets, but `-all` is implicit, while `-bar` is explicit.